src/main/java/io/mathdojo/useraccountservice/services/IdentityService.java

```diff
@@ -122,13 +124,17 @@ private boolean isValidAccountModificationRequest(AccountRequest request) {
     }
 
     public User getUserInOrg(String expectedOrganisationId, String userId) {
-        String returnedOrgId = (getOrganisationById(expectedOrganisationId)).getId();
-        if (PRECONDITIONED_UNKNOWN_USER_ID.equals(userId)) {
+    	String returnedOrgId = (getOrganisationById(expectedOrganisationId)).getId();
+        Optional<User> userToReturn = userRepo.findById(userId);
+        if (PRECONDITIONED_KNOWN_USER_ID.equals(userId)){
+        	return new User(userId, true, "", "", expectedOrganisationId);
+        }
+        if (PRECONDITIONED_UNKNOWN_USER_ID.equals(userId) || !userToReturn.isPresent() || !returnedOrgId.equals(userToReturn.get().getBelongsToOrgWithId())) {
```

Comment:
I think now that we are integrating with mongo, we should remove all the pre-conditioned known and unknown users as it could bite us later.

Reply:
theres quite a few tests that depend on these being there. How do you suggest we handle those

Reply:
Sure, you mean the unit tests or the integration tests? If integration test, the comment I made on the PR as a whole explains a possible approach. We could also try a similar approach with our unit tests. I'll also be looking into whether there are any "in memory" Mongo solutions we can plug in for unit testing.